autoload/vundle/installer.vim

```diff
@@ -6,8 +6,10 @@
 " ...    -- any number of bundle specifications (separate arguments)
 " ---------------------------------------------------------------------------
 func! vundle#installer#new(bang, ...) abort
-  let bundles = (a:1 == '') ?
+  let bundles = (a:0 == 0) ?
         \ g:bundles :
+        \ (a:bang) ?
```

Comment:
I find the nested ternary operator generally unreadable, could you maybe break it down with ifs